moonstone/normalization/reads/read_downsize.py

```diff
@@ -0,0 +1,69 @@
+import logging
+import random
+
+from moonstone.normalization.reads.base import BaseDownsizing
+
+logger = logging.getLogger(__name__)
+
+
+class DownsizePair(BaseDownsizing):
+    """Normalization for the purposes of assessing diversity. Reads are downsized by random selection of raw reads
+    generating a subset from which alpha diversity can be calculated.
+    Note that removal of data, while useful for diversity assessment, is no longer considered good practice.
+    https://doi.org/10.1371/journal.pcbi.1003531
+    """
+
+    def __init__(self, raw_f, raw_r, n=1000, seed=623):
+        """Paired reads assumes forward and reverse FASTQ files.
+        n is the number of reads that will be randomly picked, with a default of 1000.
+        A random seed is preset to 62375 to allow for reproducibility"""
+
+        super().__init__(raw_f, raw_r)
+        self.raw_f = raw_f
```

Comment:
I think you are doing this already in the `BaseDownsizing` (as well as the following line). So you can normally get rid of it.

Reply:
I removed the redundant lines.